standard-app/templates/configs/externalsecret.yaml

```diff
@@ -22,14 +22,14 @@ spec:
     creationPolicy: Owner
   data:
   {{- range $secret := $appConfig.secrets }}
-  - secretKey: {{ $secret }}
+  - secretKey: {{ $secret.secretKey }}
     remoteRef:
       {{- if eq $.Values.externalSecret.type "gcp" }}
       key: {{ $.Release.Name | upper }}_{{ $secret }}
       {{- end }}
       {{- if eq $.Values.externalSecret.type "vault" }}
       key: {{ $.Values.externalSecret.secretPath }}/{{ $.Release.Name }}
-      property: {{ $secret }}
+      property: {{ $secret.property }}
```

Comment:
It would be better if you look for the existence of `$secret.property` and use it. If it doesn't exist, use `$secret.secretKey`.  This makes the library backwards compatible and also lets values be shorter when the key and property is the same (which will be most of the time).